python/instrumentation/openinference-instrumentation-beeai/src/openinference/instrumentation/beeai/processors/tool.py

```diff
@@ -31,10 +31,6 @@ def __init__(self, event: "RunContextStartEvent", meta: "EventMeta") -> None:
             {
                 SpanAttributes.TOOL_NAME: tool.name,
                 SpanAttributes.TOOL_DESCRIPTION: tool.description,
-                # TODO: what's the difference?
-                SpanAttributes.TOOL_PARAMETERS: stringify(
-                    safe_dump_model_schema(tool.input_schema)
-                ),
```

Comment:
same here

Reply:
I would leave - it's more of a nice to have but it let's you cross compare the parameters against the actual input.